src/DetailsView/components/issues-table.tsx

```diff
@@ -112,32 +94,24 @@ export class IssuesTable extends React.Component<IssuesTableProps, IssuesTableSt
 
     private renderExportButton(): JSX.Element {
         const shouldShowButton = this.props.issuesEnabled && !this.props.scanning;
+        const { deps, scanResult, reportGenerator, pageTitle, pageUrl } = this.props;
+        const scanDate = new Date(scanResult.timestamp);
```

Comment:
should this be a dep? (I think I saw a `dateProvider` being use somewhere around our report related code)